tests/integration-tests/tests/storage/test_ebs.py

```diff
@@ -110,22 +113,40 @@ def test_ebs_multiple(scheduler, pcluster_config_reader, clusters_factory, regio
     volume_ids = get_ebs_volume_ids(cluster, region)
     for i in range(len(volume_ids)):
         # test different volume types
-        volume_type = _get_ebs_settings_by_name(cluster.config, f"ebs{i+1}")["VolumeType"]
-        volume = describe_volume(volume_ids[i], region)
+        volume_id = volume_ids[i]
+        ebs_settings = _get_ebs_settings_by_name(cluster.config, f"ebs{i+1}")
+        volume_type = ebs_settings["VolumeType"]
+        volume = describe_volume(volume_id, region)
         assert_that(volume[0]).is_equal_to(volume_type)
+        encrypted = ebs_settings.get("Encrypted")
+        if encrypted is None:
+            # Default encryption if not specified
+            encrypted = True
+        _test_ebs_encrypted_with_kms(volume_id, region, encrypted=encrypted, kms_key_id=ebs_settings.get("KmsKeyId"))
         # test different iops
         # only the iops of io1 and io2 can be configured by us
         if volume_type in ["io1", "io2", "gp3"]:
-            volume_iops = _get_ebs_settings_by_name(cluster.config, f"ebs{i+1}")["Iops"]
+            volume_iops = ebs_settings["Iops"]
             assert_that(volume[1]).is_equal_to(int(volume_iops))
 
+    _test_root_volume_encryption(cluster, os, region, scheduler, encrypted=False)
+
 
 def _get_ebs_settings_by_name(config, name):
     for shared_storage in config["SharedStorage"]:
         if shared_storage["Name"] == name:
             return shared_storage["EbsSettings"]
 
 
+def _get_root_volume_id(instance_id, os, region):
```

Comment:
This seems the same function we have in `test_tag_propagation`, let's put them in common.

Reply:
Good catch! Done!